utils/update-next.js

```diff
@@ -9,7 +9,9 @@ function exec(cmd, args, opts) {
 module.exports = async ({ github, context }) => {
   const oldVersion = require('../examples/nextjs/package.json').dependencies
     .next;
-  const newVersion = exec('npm', ['view', 'next', 'dist-tags.latest']);
+  const newVersion = exec('npm', ['view', 'next', 'dist-tags.latest'])
+    .toString()
+    .trim();
   const branch = `next-${newVersion.replaceAll('.', '-')}`;
```

Comment:
Do we want to log the output from `exec` for `create-next-app` in case it fails at all so we can debug it easier?

Reply:
Good point! Added in [8881207](https://github.com/vercel/vercel/pull/8452/commits/8881207c0e29606097c2cc9efb088e455ee3b6bf)